packages/datadog-instrumentations/src/pg.js

```diff
@@ -40,9 +40,8 @@ function wrapQuery (query) {
     const callbackResource = new AsyncResource('bound-anonymous-fn')
     const asyncResource = new AsyncResource('bound-anonymous-fn')
     const processId = this.processID
-
     return asyncResource.runInAsyncScope(() => {
-      startCh.publish({ params: this.connectionParameters, query: pgQuery, processId })
+      startCh.publish({ params: this.connectionParameters, statement: pgQuery.text, query: pgQuery, processId })
```

Comment:
FYI @crysmags, if this PR lands you'll want to follow the same pattern for mysql

Reply:
I do wonder if the names are going to confuse people. Is `statement` linguistically equivalent to `query`? Like is a SQL query and a SQL statement the same thing? If so we should probably rename the fields. Maybe `statement` should instead be `originalQuery`.

Reply:
I really didn't think a lot about the name of the property... I've used `statement` because it was the name that it had two weeks ago, in this line. I only restored the property.
But I'm agree with you, `originalQuery` or `originalSql` are better names

Reply:
I changed it to `originalQuery`